api/client/swarm/opts.go

```diff
@@ -30,9 +30,11 @@ const (
 
 var (
 	defaultPolicies = []swarm.Policy{
-		{Role: WORKER, Autoaccept: true},
-		{Role: MANAGER, Autoaccept: false},
+		{Role: worker, Autoaccept: true},
+		{Role: manager, Autoaccept: false},
 	}
+
+	exists = struct{}{}
```

Comment:
@aaronlehmann heh, okay. I still think that readability of bool version is much better and I wouldn't prefer `struct{}` version for one-time set usage.